### PR TITLE
New package: GPXSee

### PR DESCRIPTION
**Pull request by 0-st**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture:
  - x86_64-libc
- I crossbuilt this PR locally for these architectures:
  - aarch64-musl
  - armv7l
  - armv6l-musl

### CODE REVIEW

**Comment by 0-st** [comment]
Closes #36358

**Comment by paper42** [comment]
continuation of #26675. Please force push to an existing PR instead of creating a new one.

**Comment by 0-st** [comment]
> continuation of #26675. Please force push to an existing PR instead of creating a new one.

I did but github wouldn't recognise the branch anymore.

**Comment by paper42** [comment]
please squash the commits

File: srcpkgs/GPXSee/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+revision=1
+build_style=qmake
+hostmakedepends="qt5-qmake qt5-host-tools"
+makedepends="qt5-tools-devel qt5-location-devel qt5-svg-devel"
+depends="qt5-imageformats qt5-location"
+short_desc="GPS log file viewer and analyzer"
+maintainer="ghost <gh0st@vivaldi.net>"
+license="GPL-3.0-only"
+homepage="https://www.gpxsee.org/"
+distfiles="https://github.com/tumic0/${pkgname}/archive/${version}.tar.gz"
```
Comment (Duncaen) [inline srcpkgs/GPXSee/template:13]: Use the name instead of the `pkgname` variable here, its not really variable.

**Comment by paper42** [comment]
Please squash the commits every time you push to this branch, use one commit per package.

**Pull request closed without merging**